contrib/platform/src/com/sun/jna/platform/win32/GDI32.java

```diff
@@ -343,4 +353,154 @@ HBITMAP CreateDIBSection(HDC hDC, BITMAPINFO pbmi, int iUsage,
      * @return true if successful
      */
     public boolean SetPixelFormat(HDC hdc, int iPixelFormat, PIXELFORMATDESCRIPTOR.ByReference ppfd);
+
+	/**
+	 * Retrieves information for the specified graphics object.
+	 * 
+	 * @param hgdiobj
+	 *            A handle to the graphics object of interest. This can be a
+	 *            handle to one of the following: a logical bitmap, a brush, a
+	 *            font, a palette, a pen, or a device independent bitmap created
+	 *            by calling the {@link #CreateDIBSection} function.
+	 * @param cbBuffer
+	 *            The number of bytes of information to be written to the
+	 *            buffer.
+	 * @param lpvObject
+	 *            A pointer to a buffer that receives the information about the
+	 *            specified graphics object.
+	 *            <p/>
+	 *            The following table shows the type of information the buffer
+	 *            receives for each type of graphics object you can specify with
+	 *            hgdiobj.
```

Comment:
You're right! The JavaDoc HTML generation script prints several warnings about that. It seems that I'm not the only one who used HTML content in JavaDoc. I could remove all paragraph/linebreak tags but what should I do with HTML tables? Some documentations have tables?